revproxy/transformer.py

```diff
@@ -42,47 +44,62 @@ class DiazoTransformer(object):
     def __init__(self, request, response):
         self.request = request
         self.response = response
+        self.log = logging.getLogger('revproxy.transformer')
+        self.log.info("DiazoTransformer created")
 
     def should_transform(self):
-        """Determine if we should transform the response"""
+        """
+        Determine if we should transform the response
+        """
 
         if not HAS_DIAZO:
+            self.log.info("HAS_DIAZO: false")
             return False
 
         if asbool(self.request.META.get(DIAZO_OFF_REQUEST_HEADER)):
+            self.log.info("DIAZO_OFF_REQUEST_HEADER in request.META: off")
             return False
 
         if asbool(self.response.get(DIAZO_OFF_RESPONSE_HEADER)):
+            self.log.info("DIAZO_OFF_RESPONSE_HEADER in response.get: off")
             return False
 
         if self.request.is_ajax():
+            self.log.info("Request is AJAX")
             return False
 
         if self.response.streaming:
+            self.log.info("Response has streaming")
             return False
 
         content_type = self.response.get('Content-Type')
         if not is_html_content_type(content_type):
+            self.log.info("Content-type: false")
             return False
 
         content_encoding = self.response.get('Content-Encoding')
         if content_encoding in ('zip', 'compress'):
+            self.log.info("Content encode is zip or compress")
```

Comment:
In the cases where the variable could have more than one value I'd log it. For example in this case:
self.log.info("Content-Encoding: %s", content_encoding)